docs/UserGuide.md

```diff
@@ -323,14 +329,15 @@ Example:
 
 #### 3.3.2 Removing a student's deadline: `delete -d`
 
-Removes a deadline assigned to a student specified by ID & a priority rank. The specified deadline must exist in the student's deadline list previously.
+Removes a deadline assigned to a student specified by ID & a priority rank (Below highlighted task has a priority rank of 2). The specified deadline must exist in the student's deadline list previously.
 
 Format: `delete -d i/STUDENT_ID r/DEADLINE_RANK`
 
 Example:
 
 * `delete -d i/A0123456G r/1`
-
+   ![Priority](images/Priority.png) <br> <br>
```

Comment:
Indentation doesn't match, user might think this belongs to the previous example